scripts/download-oas.sh

```diff
@@ -24,5 +24,31 @@ cd ${work_dir}
 git clone ${OAS_REPO}
 
 for service_dir in ${work_dir}/${OAS_REPO_NAME}/services/*; do
-    mv -f ${service_dir}/*.json ${ROOT_DIR}/oas
+    max_version_dir=""
+    max_version=""
+    max_version_is_beta=false
+
+    for dir in ${service_dir}/*; do
+        version=$(basename "$dir")
+        current_version_is_beta=false
+        # Check if directory name starts with 'v'
+        if [[ ${version} == v* ]]; then
+            # Remove the 'v' prefix
+            version=${version#v}
+            # Check if version is beta
+            if [[ ${version} == *beta* ]]; then
+                # Remove 'beta' suffix
+                version=${version%beta*}
+                current_version_is_beta=true
+            fi
+            # Compare versions and prioritize non-beta versions
+            if [[ ${version} > ${max_version} || (${version} == ${max_version} && ${current_version_is_beta} == false) ]]; then
```

Comment:
Did you test if this works? ${version} is technically a string, so this may not work as expected

Reply:
Fixed, it was not working for cases like 9 < 12